sonic_platform_base/sonic_xcvr/api/public/sff8472.py

```diff
@@ -295,3 +295,34 @@ def get_lpmode_support(self):
 
     def get_power_override_support(self):
         return False
+
+    def is_active_cable(self):
+        return self.xcvr_eeprom.read(consts.SFP_CABLE_TECH_FIELD) == 'Active Cable'
+
+    def get_overall_offset(self, page, offset, size, wire_addr=None, flat=False):
+        if not wire_addr:
+            raise ValueError("Invalid wire address for sff8472, must a0h or a2h")
+
+        is_active_cable = self.is_active_cable()
+        valid_wire_address = ('a0h', 'a2h') if is_active_cable else ('a0h',)
```

Comment:
For transceivers such as 10G-SR which has page a2h, we will treat page a2h as an invalid option. Should we document this?

Reply:
We mentioned this in HLD.